frmts/tiledb/tiledbdense.cpp

```diff
@@ -2933,7 +2933,9 @@ CPLErr TileDBRasterDataset::IBuildOverviews(
             {
                 CPL_IGNORE_RET_VAL(poODS->Close());
                 tiledb::Array::delete_array(*m_ctx, poODS->GetDescription());
-                vfs.remove_dir(poODS->GetDescription());
+                if (vfs.is_dir(poODS->GetDescription()) {
```

Comment:
```suggestion
                if (vfs.is_dir(poODS->GetDescription())) {
```

can you also include a revert of 733a50e5629efaa3892d8803f1f0d61bc321e6b6 in your PR? Thanks